skiko/src/commonMain/kotlin/org/jetbrains/skia/icu/Unicode.kt

```diff
@@ -0,0 +1,43 @@
+package org.jetbrains.skia.icu
+
+import org.jetbrains.skia.ExternalSymbolName
+import org.jetbrains.skia.impl.Library.Companion.staticLoad
+
+object CharDirection {
+    init {
+        staticLoad()
```

Comment:
why do we need staticLoad() ?

Reply:
In case of JNI we need to make sure that library is loaded before calling any `external` methods. To see what problem exactly it cause (missing method to call bla bla) - comment it and run single test from `UnicodeTest`